thirdeye-ui/src/app/components/parse-markdown/parse-markdown.component.tsx

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2022 StarTree Inc
+ *
+ * Licensed under the StarTree Community License (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at http://www.startree.ai/legal/startree-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT * WARRANTIES OF ANY KIND,
+ * either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { Box, Typography } from "@material-ui/core";
+import React, { FunctionComponent } from "react";
+import {
+    ReactMarkdown,
+    ReactMarkdownOptions,
+} from "react-markdown/lib/react-markdown";
+import { LinkV1 } from "../../platform/components";
+import type { ParseMarkdownProps } from "./parse-markdown.interfaces";
+
+export const ParseMarkdown: FunctionComponent<ParseMarkdownProps> = ({
+    children,
+    customOptions,
+}) => {
+    const options: ReactMarkdownOptions = {
+        components: {
+            h1: ({ children }) => (
+                <Typography variant="h1">{children}</Typography>
+            ),
+            h2: ({ children }) => (
+                <Typography variant="h2">{children}</Typography>
+            ),
+            h3: ({ children }) => (
+                <Typography variant="h3">{children}</Typography>
+            ),
+            h4: ({ children }) => (
+                <Typography variant="h4">{children}</Typography>
+            ),
+            h5: ({ children }) => (
+                <Typography variant="h5">{children}</Typography>
+            ),
+            h6: ({ children }) => (
+                <Typography variant="h6">{children}</Typography>
+            ),
+            a: ({ children, href, ...props }) => (
+                <LinkV1
+                    {...props}
+                    color="primary"
+                    href={`//${href}`}
```

Comment:
What's the `//` for?

Reply:
The links are all opening as relative links. Even when there is an `https://` present in the link, it opens like `http://localhost:7004/alerts/create/new/simple/https://www.google.com`

Is this `//` fix fine, or should I look into using a different link component?

Reply:
Try passing `externalLink` to see if it still gives you issues like

```
<LinkV1
                    {...props}
                    externalLink
                    color="primary"
```

Reply:
Added `externalLink`